common/main/root.xml

```diff
@@ -5260,6 +5260,9 @@ Warnings: All cp values have U+FE0F characters removed. See /annotationsDerived/
 			<currency type="RWF">
 				<symbol alt="narrow">RF</symbol>
 			</currency>
+			<currency type="SAR" alt="variant">
+    			<symbol alt="variant">&#20C1;</symbol>
```

Comment:
```suggestion
    			<symbol alt="variant">&#x20C1;</symbol>
```
see https://en.wikipedia.org/wiki/Numeric_character_reference

Reply:
Thanks, I thought it looked odd...